app/models/user.rb

```diff
@@ -39,6 +39,11 @@ def github_client_scopes
     GitHubUser.new(github_client, uid).client_scopes
   end
 
+  # for searching, disable pagination so we can control the number of suggestions
+  def github_search_client
```

Comment:
@johndbritton that sound fine to me

@cyhsutw if you could memoize the `seach_client` that'd be ✨ 

```ruby
def search_client
  @search_client ||= Octokit::Client.new(access_token: current_user.token, auto_paginate: false)
end
```